oem/ibm/libpldmresponder/file_io_type_lid.hpp

```diff
@@ -169,12 +168,39 @@ class LidHandler : public FileHandler
                     oemIbmPlatformHandler = dynamic_cast<
                         pldm::responder::oem_ibm_platform::Handler*>(
                         oemPlatformHandler);
+                // rc = validate api;
+                auto& bus = pldm::utils::DBusHandler::getBus();
                 auto sensorId =
                     oemIbmPlatformHandler->codeUpdate->getMarkerLidSensor();
                 using namespace pldm::responder::oem_ibm_platform;
+                auto markerLidName = "80a00001.lid";
+                auto markerLidDirPath =
+                    fs::path(LID_STAGING_DIR) / "lid" / markerLidName;
+                try
+                {
+                    rc = processCodeUpdateLid(lidPath);
+                    if (rc != PLDM_SUCCESS)
+                    {
+                        return rc;
+                    }
+                    auto method = bus.new_method_call(
+                        "xyz.openbmc_project.Software.BMC.Updater",
+                        "/xyz/openbmc_project/software",
+                        "xyz.openbmc_project.Software.LID", "Validate");
+                    method.append(markerLidDirPath.c_str());
+                    bus.call_noreply(method);
+                }
+                catch (const sdbusplus::exception::exception& e)
```

Comment:
We want to catch the individual errors that the Validate interface throws, at least the entitlement and the Incompatible one which is the min mif level (same for the validate call below): 
https://github.com/ibm-openbmc/phosphor-dbus-interfaces/blob/9e480210e999b3e090129e5849c1de8cdce03e35/yaml/xyz/openbmc_project/Software/LID.interface.yaml#L14

catch (ExpiredKey)
    ENTITLEMENT_FAIL
catch (Incompatible)
    MIN_MIF_FAIL